lib/utils/http-requestor.js

```diff
@@ -60,6 +73,18 @@ class HttpRequestor extends BaseRequestor {
       if (u.port) this.client = new Client(`${u.protocol}://${u.resource}:${u.port}`);
       else this.client = new Client(`${u.protocol}://${u.resource}`);
     }
+
+    if (NODE_ENV == 'test' && process.env.JAMBONES_HTTP_PROXY_IP) {
```

Comment:
why is this in test mode only?

Reply:
In production: setGlobalDispatcher will be initiated only 1 time when feature server is boot.

When running in our testsuite, these HTTP PROXY env variables will be set on and off and we want to change global dispatcher whenever these env variables is changed.